apps/files/src/components/FileSharingSidebar.vue

```diff
@@ -297,8 +297,9 @@ export default {
       return this.highlightedFile.canShare()
     },
     noResharePermsMessage() {
-      const translated = this.$gettext("You don't have permission to share this %{type}.")
-      return this.$gettextInterpolate(translated, { type: this.highlightedFile.type }, false)
+      const translatedFile = this.$gettext("You don't have permission to share this file.")
```

Comment:
why not also use "resource" here and everywhere we had "file" and "folder" as well like for below ?

Reply:
I honestly don't care much - this is a design/product decision you have to take as a team

I'm just fixing bugs in the existing code base - with minimal invasive methodology :shrug: